test/kubernetes/e2e/features/client_tls/types.go

```diff
@@ -1,68 +1,53 @@
 package client_tls
 
 import (
+	_ "embed"
 	"net/http"
-	"path/filepath"
 
 	"github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	kubev1 "github.com/solo-io/gloo/projects/gateway/pkg/api/v1/kube/apis/gateway.solo.io/v1"
 	"github.com/solo-io/gloo/test/gomega/matchers"
-	"github.com/solo-io/skv2/codegen/util"
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var (
-	annotatedNginxSvcManifestFile       = filepath.Join(util.MustGetThisDir(), "testdata", "annotated-nginx-svc.yaml")
-	annotatedNginxOneWaySvcManifestFile = filepath.Join(util.MustGetThisDir(), "testdata", "annotated-oneway-nginx-svc.yaml")
-	nginxUpstreamManifestFile           = filepath.Join(util.MustGetThisDir(), "testdata", "nginx-upstream.yaml")
-	nginxOneWayUpstreamManifestFile     = filepath.Join(util.MustGetThisDir(), "testdata", "nginx-oneway-upstream.yaml")
-	tlsSecretManifestFile               = filepath.Join(util.MustGetThisDir(), "testdata", "tls-secret.yaml")
-	vsTargetingKubeManifestFile         = filepath.Join(util.MustGetThisDir(), "testdata", "vs-targeting-kube.yaml")
-	vsTargetingUpstreamManifestFile     = filepath.Join(util.MustGetThisDir(), "testdata", "vs-targeting-upstream.yaml")
+//go:embed testdata/nginx-upstreams.yaml
```

Comment:
Is this the new preferred way of pulling in manifests? If so, is there a way to share that this is the preferred approach across teams/repos?

Reply:
Great point. Ill do the following:
- Update the base testing suite, as an example 
- Add a readme with best practices and list this explicitly

How does that sound? I could also update our example https://github.com/solo-io/gloo/tree/main/test/kubernetes/e2e/example which ive tried to have be the go to "standard" so people can try to copy it